modules/View.js

```diff
@@ -86,12 +87,26 @@
      * @method render
      */
     render: function() {
-      this.unplug();
```

Comment:
good question and valid point. I've had to go back a few times and update the list view to mirror a normal view. Not easy to refactor but I can look into it.